Tests/Cells/BaseBrickCellTests.swift

```diff
@@ -80,7 +80,37 @@ class BaseBrickCellTests: XCTestCase {
         XCTAssertEqual(cell?.topSeparatorLine.backgroundColor, .blueColor())
         XCTAssertNil(cell?.bottomSeparatorLine.superview)
     }
-
+    
+    func testSectionFixedHeight() {
```

Comment:
What would happen if the bricks inside a section would exceed the fixed height of a section with a fixed height? I'd like to see a test for that